### DISCUSSION TITLE
[UDISE] Color coding is not visible in map of India

### DISCUSSION THREAD

**Discussion by Mohanapriya777** [Sunbird-cQube/community#828] [General]
Issue: (cQube Puducherry) Colour coding of district as per legends is not visible in map of India
Program: UDISE

![screenshot](https://github.com/Sunbird-cQube/community/assets/130358614/f51e1648-f25e-4cc2-90bd-1593cd48b3ec)

**Comment by RNarmatha** [comment]
@bommalla, @pandutibil - Please look into this issue

**Comment by pandutibil** [comment]
@Mohanapriya777 could you please provide the sample data format that you're sending it to the map report here?

**Reply by Mohanapriya777** [reply]
Here i have attached the UDISE data. Kindly check this.
[udise_program-started.csv](https://github.com/Sunbird-cQube/community/files/12924922/udise_program-started.csv)
[udise-all-dashboard.csv](https://github.com/Sunbird-cQube/community/files/12924925/udise-all-dashboard.csv)

**Reply by pandutibil** [reply]
Okay!

**Reply by Mohanapriya777** [reply]
Hi @pandutibil any update on this.

**Comment by pandutibil** [comment]
@Mohanapriya777 Which version of cQube you're using?

**Reply by Mohanapriya777** [reply]
@pandutibil V5.0.3

**Reply by pandutibil** [reply]
We are checking on it, we'll get back to you @Mohanapriya777 
Thank You

**Reply by Mohanapriya777** [reply]
Hi @pandutibil The issue is resolved. We rectified the issue in data and it's working now. Thank you.

**Reply by pandutibil** [reply]
@Mohanapriya777 is that issue resolved?

**Reply by Mohanapriya777** [reply]
Yes

**Reply by pandutibil** [reply]
What went wrong?

**Reply by Mohanapriya777** [reply]
Latitude & longitude is wrong in dimension file.We changed and uploaded the new files now.

**Reply by pandutibil** [reply]
Okay got it
